.buildkite/build.rayci.yml

```diff
@@ -28,6 +28,7 @@ steps:
       - forge
 
   - label: ":tapioca: build: jar"
+    key: java_wheels
```

Comment:
does java tests actually fail or matter very much? I thought we do not really have much of java related dev activities now.

Reply:
i noticed they fail in this PR here https://buildkite.com/ray-project/premerge/builds/26587; all the wheels fail actually but it's cheap enough so i add it